test/e2e/v1beta1/validators_test.go

```diff
@@ -213,10 +213,13 @@ func validateBuildRunResultsFromBundleSource(testBuildRun *buildv1beta1.BuildRun
 func validateBuildRunToFail(testBuild *utils.TestBuild, testBuildRun *buildv1beta1.BuildRun) {
 	trueCondition := corev1.ConditionTrue
 	falseCondition := corev1.ConditionFalse
+	var err error
 
 	// Ensure the BuildRun has been created
-	err := testBuild.CreateBR(testBuildRun)
-	Expect(err).ToNot(HaveOccurred(), "Failed to create BuildRun")
+	if _, err := testBuild.GetBR(testBuildRun.Name); err != nil {
```

Comment:
I do not understand that check. Why do we need to check if the BuildRun exists ?

Reply:
Because in my tests, I have already created the buildrun, otherwise I had to create a new function for running buildruns.
This condition is also present in `validateBuildRunToSucceed` function.